geoh/geoh.py

```diff
@@ -33,9 +49,15 @@ def geohashes_polygon_intersection(polygon, geohashes=[]):
     """Return geohashes that intersect `polygon`"""
     from itertools import compress
     geoms = map(_geohash_to_shape, geohashes)
-    intersections = map(lambda gh: gh.intersects(polygon), geoms)
+    intersections = map(lambda g_h: g_h.intersects(polygon), geoms)
     return list(compress(geohashes, intersections))
 
+def geohashes_polygon_within(polygon, geohashes=[]):
+    from itertools import compress
+    geoms = map(_geohash_to_shape, geohashes)
+    withins = map(lambda g_h: g_h.within(polygon), geoms)
+    return list(compress(geohashes, withins))
```

Comment:
Could we add a simple docstring to this function?

Reply:
Sure. Added `"""Return geohashes that are within the polygon"""`.